packages/react-magma-dom/src/components/Stepper/Step.tsx

```diff
@@ -0,0 +1,242 @@
+      return props.theme.colors.neutral100;
+    }
+    return props.theme.colors.primary600;
+  } else {
+    return props.theme.colors.neutral100;
+  }
+}
+
+const StyledStep = styled.div<StepProps>`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  align-self: self-start;
+`;
+
+const StyledStepIndicator = styled.span<StepProps>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 50%;
+  width: 24px;
+  height: 24px;
+  box-shadow: inset 0 0 0 2px
+    ${props =>
+      (props.stepStatus === 'incomplete' && !props.hasError) ||
+      (props.stepStatus === 'active' && !props.hasError)
+        ? StepCircleOutlineColors
+        : 'none'};
+  background: ${StepCircleBackgroundColors};
+  svg {
+    color: ${StepSvgColors};
+    width: ${props => props.theme.spaceScale.spacing05};
+    height: ${props => props.theme.spaceScale.spacing05};
+  }
+`;
+
+const StyledStepTextWrapper = styled.div<StepProps>`
+  flex: 1;
+  display: flex;
+  align-self: center;
+  flex-direction: column;
+  position: relative;
+  margin: 6px -10em 0;
+`;
+
+const StyledLabel = styled.span<StepProps>`
+  color: ${StepLabelColors};
+  font-size: ${props => props.theme.typeScale.size02.fontSize};
+  font-weight: 600;
+  line-height: normal;
+`;
+
+const StyledSecondaryLabel = styled.span<StepProps>`
+  color: ${StepLabelColors};
+  font-size: ${props => props.theme.typeScale.size01.fontSize};
+  line-height: normal;
+  margin: 5px 0;
+`;
+
+export const Step = React.forwardRef<HTMLDivElement, StepProps>(
+  (props, ref) => {
+    const {
+      breakpointStyle,
+      children,
+      currentStep,
+      hasError,
+      hideLabels,
+      summaryView,
+      label,
+      secondaryLabel,
+      testId,
+      isInverse: isInverseProp,
+      stepStatus,
+      ...rest
+    } = props;
+    const theme = React.useContext(ThemeContext);
+    const isInverse = useIsInverse(isInverseProp);
+
+    return (
+      <StyledStep
+        currentStep={currentStep}
+        theme={theme}
+        isInverse={isInverse}
+        ref={ref}
+        data-testid={testId}
+        {...rest}
+      >
+        {stepStatus && (
+          <StyledStepIndicator
+            hasError={hasError}
+            isInverse={isInverse}
+            stepStatus={stepStatus}
+            theme={theme}
+          >
+            {stepStatus === 'complete' && !hasError && <CheckIcon />}
+            {hasError && <CrossIcon />}
+          </StyledStepIndicator>
+        )}
+        <StyledStepTextWrapper theme={theme} isInverse={isInverse} tabIndex={0}>
+          {!summaryView && !hideLabels ? (
+            <>
+              {label && (
+                <StyledLabel label={label} isInverse={isInverse} theme={theme}>
+                  {label}
+                </StyledLabel>
+              )}
+              {secondaryLabel && (
+                <StyledSecondaryLabel
+                  secondaryLabel={secondaryLabel}
```

Comment:
Same here, this isn't necessary

Reply:
Needed for styling: 

`function stepLabelColors(props: StepProps) {
  if (props.isInverse) {
    if (props.label) {
      return props.theme.colors.neutral100;
    } else if (props.secondaryLabel) {
      return transparentize(0.3, props.theme.colors.neutral100);
    }
  } else {
    if (props.label) {
      return props.theme.colors.neutral700;
    } else if (props.secondaryLabel) {
      return props.theme.colors.neutral500;
    }
  }
}`

Reply:
You can do something like 
`<StyledSecondaryLabel isInverse={isInverse} theme={theme}>{secondaryLabel}</StyledSecondaryLabel>`
then
```
function stepLabelColors(StepProps, hasLabel, hasSecondaryLabel){}
```
and
```
const StyledSecondaryLabel = styled.span<StepProps>`
  color: ${stepLabelColors(props, false, true)};
  font-size: ${props => props.theme.typeScale.size01.fontSize};
  line-height: normal;
  margin: 5px 0;
`;
const StyledLabel = styled.span<StepProps>`
  color: ${stepLabelColors(props, true, false)};
  font-size: ${props => props.theme.typeScale.size02.fontSize};
  font-weight: 600;
  line-height: normal;
`;
```
since you know it's always going to be true or false in those cases without passing the label string